pinc/CharSuites.inc

```diff
@@ -178,3 +178,44 @@ foreach($charsuite_files as $charsuite_file)
 {
     include_once($charsuite_file);
 }
+
+trait CharSuiteSet
```

Comment:
I don't think this level of complexity is necessary just for quizzes. The hack in `pinc/CharacterSelector.inc` isn't great, but it's at least localized there instead of spilling out into multiple places that is only used for the quizzes.

Reply:
Perhaps I have been over-assiduous in renaming variables in various files which I will undo.

Reply:
Your variable names are all good here, I just don't think we need this level of abstraction to solve this one-off case for quizzes.